app/assets/javascripts/lessons.js

```diff
@@ -89,40 +99,53 @@ function isLessonPage() {
 }
 
 function constructLessonSections() {
-  var lessonHeadings = getElements('.lesson-content h3');
-  lessonHeadings.forEach(function(heading) {
-    const id = heading.getAttribute('id');
-    heading.removeAttribute('id');
+  lessonHeadings.forEach(buildScrollSpy);
+  uri = location.href.replace(location.hash,"");
```

Comment:
And please use single quotes to stay consistent.

Reply:
There are a few other places where single quotes should be used. I haven't commented in all those lines but please get them replaced.